doc/api/esm.md

```diff
@@ -524,8 +495,9 @@ following conditions:
 
 Write the package in CommonJS or transpile ES module sources into CommonJS, and
 create an ES module wrapper file that defines the named exports. Using
-[Conditional Exports][], the ES module wrapper is used for `import` and the
-CommonJS entry point for `require`.
+[Conditional Exports][] via the `--experimental-conditional-exports` flag, the
```

Comment:
Could we do what I had previously by having the flagging note at the top in italics - 

```
_This approach is currently experimental and only works under the `--experimental-conditional-exports` flag._
```

Reply:
I don't think so, because conditional exports no longer applies to the whole section. The last part is an example of using the non-conditional syntax.

Reply:
I just want to see a single line on its own in italics noting the flag along with the use case.

I know we say to use the flag, but still this is an instruction and piece of advice that will simply not work unflagged. So we need to be very very clear that it is a flagged feature, and highlight that as its own line along with the example.

Perhaps rather just include a line above the example like:

_All consumers of this package will need to use the `--experimental-conditional-exports` flag to ensure `require()` support_

Because otherwise, the footgun is testing it in modern Node - `import()` works, great. Then testing it in legacy Node - `require()` works, great. But forgetting that `require()` breaks in modern Node.js and only finding that out after publishing.

It needs to be documented as a footgun. Because it is.